wots.c

```diff
@@ -109,6 +111,78 @@ static void chain_lengths(const xmss_params *params,
  *
  * Writes the computed public key to 'pk'.
  */
+
+typedef struct wots_pkgen_args
+{
+    const xmss_params *params;
```

Comment:
Is there already documentation for those struct members elsewhere? If not, can you add documentation here?

Reply:
fix in https://github.com/kj3moraes/further-testing-xmss/pull/5/commits/46648c53b9aa817fef5cde8cdf7c1d2004a92b05

Reply:
I don't see the documentation in that commit.

Reply:
Oh, it should be in this commit, https://github.com/kj3moraes/further-testing-xmss/pull/5/files#diff-8abae1a1630bbc61fc3689c457ec80afbcb32ccae984fcda0db7a26b71ffce0bR121

Reply:
What I meant is this: It would be nice to have a line commenting each and every member of the structure.

Reply:
can you check in this commit: 
https://github.com/kj3moraes/further-testing-xmss/pull/5/commits/fc925eb997a4d8ce9827f174f69eedde9c6293cb